portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,27 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="header">
+        <h1>Molly Graton</h1>
+        <a href="https://www.linkedin.com/in/mollygraton" target="_blank">LinkedIn</a>
+        <a href="https://www.github.com/mollygraton/" target="_blank">GitHub</a>
+    </div>
+    <div id="aboutme"> 
+    </div>            
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <img src="/images/gcpicture.jpg" alt="picture of me" 
+        style="width:150px;height:200px;">
+      <p>This is my portfolio!</p>
+      <p>Hi! I'm Molly & I'm a rising sophomore at UIUC. In my free time,
+          I enjoy biking & playing uke.
+      </p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <div id="photography">
+    </div>
+    <div id="talents">
+    </div>        
```

Comment:
Is this intended that there is no content in between?

Reply:
Yep, was just creating the div layout for the page -- will add content in upcoming commits!